specifications/api/swagger.yaml

```diff
@@ -5077,6 +5077,19 @@ x-errExistingKeyWitnesses: &errExistingKeyWitnesses
       type: string
       enum: ['existing_key_witnesses']
 
+x-errConflictingInputResolution: &errConflictingInputResolution
+  <<: *responsesErr
+  title: conflicting_input_resolution
+  properties:
+    message:
+      type: string
+      description: |
+        Occurs when providing an input resolution (input -> output mapping) as part of the
+        `inputs` field which conflicts with the wallet UTxO set when balancing.
```

Comment:
Btw, I noticed the api docs for errors are kind of weird; the description looks specific to the `message` field rather than to the error, and I'm not sure to what extend the error code make sense:

<img width="826" alt="Skärmavbild 2022-08-30 kl  15 35 03" src="https://user-images.githubusercontent.com/304423/187451286-4015c65c-123a-4e16-b9f8-758441b97994.png">

Also `err400 bad request` seemed more appropriate than `err403 forbidden` for `unresolved_inputs` and `input_resolution_conflicts`, so I went with that.